### DISCUSSION TITLE
Metadata from an encode without litterboxes

### DISCUSSION THREAD

**Discussion by SpasV** [quietvoid/dovi_tool#329] [Q&A] [answered]
### Discussed in https://github.com/quietvoid/dovi_tool/discussions/74

<div type='discussions-op-text'>

<sup>Originally posted by SpasV Decemper 28, 2024</sup>
Can I use the metadata from an encode without litterboxes for a Remux with litterboxes.
If yes, how to edit them?
</div>

**Comment by quietvoid** [comment, marked as answer]
> Can I use the metadata from an encode without litterboxes for a Remux with litterboxes.
If yes, how to edit them?

Yes and you shouldn't need to edit them unless you want to match the video with letterboxing.
See `editor` example: https://github.com/quietvoid/dovi_tool/blob/main/assets/editor_examples/active_area.json
